eark_validator/model/manifest.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from enum import Enum, unique
+from pathlib import Path
+from typing import List
+
+from pydantic import BaseModel
+
+from eark_validator.model.checksum import Checksum
+
+class ManifestEntry(BaseModel):
+    path : Path | str
```

Comment:
This is not valid for Python3.9 so my initial attempts to use the package caused an exception due to this not being a supported use of the | operator.
Installing Python3.10 or above resolves this error. And while Python 3.10 is gaining greater support, Python3.9 is still quite prevalent and is the default Python3.x version on many Linux distributions. 
So either a minimum version requirement should be mentioned in the README file, or we make this backwards compatible with Python3.9.

There are other instances of this type hinting usage of the | character, but this was the first for me to raise an exception.

Reply:
I'm with you, I think backward compatibility is more important here and 3.11 as mandatory is too restrictive. A build matrix with the appropriate versions here would pick this up automatically.

Reply:
I'm beginning to wonder, Python 3.9 is EOL in October 2025: <https://devguide.python.org/versions/> and the code is fine with 3.10 and beyond: <https://github.com/E-ARK-Software/eark-validator/actions/runs/7758461444>

Thoughts welcome.